src/intl/types.js

```diff
@@ -67,7 +76,7 @@ export class FTLKeyword extends FTLBase {
 export class FTLList extends Array {
   toString(ctx) {
     const lf = ctx._memoizeIntlObject(
-      L20nIntl.ListFormat // XXX add this.opts
+      ListFormat // XXX add this.opts
```

Comment:
It was easier to implement `FTLList` as a subclass of `Array` for now.  This meant that we couldn't use the second argument to the constructor for `ops` because it's the second element of the array.  (It's dangerous to override the `Array`'s constructor because it is still used in other places internally, e.g. when calling `.map()`).  I think the easiest will be to add a static `FTLList.create(iterable, opts)` in plce of the currently-usd `FTLList.from` (inherited from `Array`; also can't override the second arg).  I just didn't get around to doing it.

Or we could not sublass `Array` and implement custom iteration and `.push()` and `.map()`.  or maybe re-use `Array.prototype.push` and `Array.prototype.map`;  I'd need to try.

Do we need `opts` right now or can we add them later on?